amgut/templates/new_participant.html

```diff
@@ -10,7 +87,7 @@
     }
 
     function relax_parents(box) {
-        var sec_parent = document.getElementById(box.form.id + "_parent_2_name");
+        var sec_parent = $("#" + box.form.id + "_parent_2_name");
```

Comment:
Just curious about the purpose of this change. Also, I like the name of this function :smile: